static/js/webrtc.js

```diff
@@ -552,14 +627,74 @@ var rtc = (function() {
         .catch(function(err) {self.showUserMediaError(err)})
     },
     avInURL: function() {
-      if (window.location.search.indexOf("av=YES") > -1) {
+      if (urlParamString.indexOf("av=YES") > -1) {
         return true;
       } else {
         return false;
       }
     },
+
+    // Connect a setting to a checkbox. To be called on initialization.
```

Comment:
I use this for two different settings now. I plan to use this on the `enabled` setting in a future PR.